docs/reports/unmapped.md

```diff
@@ -0,0 +1,19 @@
+# Mapping progress report
+| Ontology                           | Tot terms   | Tot excluded   | Tot deprecated   | Tot mappable _(!excluded, !deprecated)_   | Tot mapped _(mappable)_   | Tot unmapped _(mappable)_   | % unmapped _(mappable)_   |
+|:-----------------------------------|:------------|:---------------|:-----------------|:------------------------------------------|:--------------------------|:----------------------------|:--------------------------|
+| [ICD10WHO](./unmapped_icd10who.md) | 12,542      | 0              | 0                | 12,542                                    | 18                        | 12,524                      | 99.9%                     |
+| [ICD10CM](./unmapped_icd10cm.md)   | 95,847      | 15,452         | 0                | 80,395                                    | 1,160                     | 79,235                      | 98.6%                     |
+| [NCIT](./unmapped_ncit.md)         | 174,278     | 155,232        | 0                | 19,046                                    | 6,734                     | 12,312                      | 64.6%                     |
```

Comment:
Numbers for NCIT not yet accurate due to using cached `mirror_signature` which also doesn't show `deprecated`. See: #171